tests/profile/test_jsonschema_profiler.py

```diff
@@ -227,9 +227,11 @@ def test_profile_simple_schema(simple_schema):
             "meta": {},
         },
     ]
+    context = empty_data_context
+    context.save_expectation_suite(obs)
```

Comment:
All these tests now attempt to save the generated suites thereby catching this bug.